base/shared/src/main/scala/scalaz/data/KleisliSyntax.scala

```diff
@@ -0,0 +1,48 @@
+package scalaz
+package data
+
+trait KleisliSyntax {
+  import Kleisli.{ runKleisli, wrapKleisli }
+
+  implicit class ToKleisliOps[F[_], A, B](k: Kleisli[F, A, B]) {
```

Comment:
we should figure out how to zero-cost this, but not now

Reply:
mosdef